### DISCUSSION TITLE
I have linked Firefly III and GoCardless  via importer tool. How to ensure live transactions are getting updated?

### DISCUSSION THREAD

**Discussion by dageres** [firefly-iii/firefly-iii#8897] [Firefly III]
I have linked Firefly III and GoCardless  via importer tool. How to ensure live transactions are getting updated?

**Comment by JC5** [comment]
I'm not sure what "live transactions" are, but the data importer does not update existing transactions.

**Comment by dageres** [comment]
Is there a way to automatically update new transactions?

**Reply by digitlength** [reply]
https://docs.firefly-iii.org/how-to/data-importer/import/automated/

**Reply by dageres** [reply]
![image](https://github.com/firefly-iii/firefly-iii/assets/20250011/5adedfd2-0da7-4c71-8771-74ff70235ce0)

**Comment by dageres** [comment]
I followed the instructions 100% and got an error 
![image](https://github.com/firefly-iii/firefly-iii/assets/20250011/caf689ca-1878-4e7a-980d-eee1d4cd94d8)

**Reply by JC5** [reply]
Tell me about the `config.json`-file which is absolutely present in the directory you mounted under `/import`.

**Comment by dageres** [comment]
{
    "version": 3,
    "source": "ff3-importer-1.5.2",
    "created_at": "2024-05-20T19:54:08+02:00",
    "date": "",
    "default_account": 1,
    "delimiter": "comma",
    "headers": false,
    "rules": true,
    "skip_form": true,
    "add_import_tag": true,
    "roles": [],
    "do_mapping": [],
    "mapping": [],
    "duplicate_detection_method": "classic",
    "ignore_duplicate_lines": false,
    "unique_column_index": 0,
    "unique_column_type": "external-id",
    "flow": "nordigen",
    "content_type": "csv",
    "custom_tag": "",
    "identifier": "0",
    "connection": "0",
    "ignore_spectre_categories": false,
    "grouped_transaction_handling": "",
    "use_entire_opposing_address": false,
    "map_all_data": true,
    "accounts": {
        "123": 1
    },
    "date_range": "all",
    "date_range_number": 30,
    "date_range_unit": "d",
    "date_not_before": "",
    "date_not_after": "",
    "nordigen_country": "GB",
    "nordigen_bank": "HSBC_HBUKGB4B",
    "nordigen_requisitions": {
        "XXXXXXXXXX": "XXXXX"
    },
    "nordigen_max_days": "90",
    "conversion": false,
    "ignore_duplicate_transactions": true
}

On Mon, 20 May 2024 at 05:53, James Cole ***@***.***> wrote:

> Tell me about the config.json-file which is absolutely present in the
> directory you mounted under /import.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/firefly-iii/discussions/8897#discussioncomment-9492325>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE2P3GYVCNKNBHWSLKVRQWTZDF6TLAVCNFSM6AAAAABH5ZLR36VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TIOJSGMZDK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Regards,

Dags Hofrats
07506964370

**Reply by JC5** [reply]
Sorry friend, but this isn't a game of "20 questions". I'm trying to make you share your work process and Firefly III setup but I can't help you if we go about it one step at a time. 

Explain to me how you got to this point and what you need to get this fixed. I'm doing this in my free time, so please stop treating me like some service desk employee because that's how it's starting to feel.

**Reply by dageres** [reply]
Sorry. I am new to self hosting things.
The whole point is to is to automate imports via gocardless via cron job.
I followed the instructions https://docs.firefly-iii.org/how-to/data-importer/import/automated/ and got that error. How to fix it ?
